### PR TITLE
Update Docker and CI status tables and content

### PR DESCRIPTION
**Pull request by jamoralp**

### CODE REVIEW

**Review by FranFin — DISMISSED** [review]

File: _docs/overview/docker_ci_status/index.md [hunk 1]
```diff
@@ -1,52 +1,78 @@
...
+  * Crazyflie drone demo: [crazyflie_demo](https://github.com/micro-ROS/micro-ROS_crazyflie_demo)
+  * Collection of generic demos, using publishers, subscribers, custom messages, services and graph introspection capabilities: [micro-ROS-demos](https://github.com/micro-ROS/micro-ROS-demos)
+  * Demo applications for FreeRTOS: [freertos_apps](https://github.com/micro-ROS/freertos_apps)
+  * Demo applications for Zephyr: [zephyr_apps](https://github.com/micro-ROS/zephyr_apps)
+  * Demo applications for NuttX (discontinued): [nuttx_apps](https://github.com/micro-ROS/nuttx_apps)
+  * Demo applications for Raspbian: [raspbian_apps](https://github.com/micro-ROS/raspbian_apps)
 * Client library:
-  * Extensions to rcl, i.e. the ROS 2 C API: [rcl_executor](https://github.com/micro-ROS/rcl_executor), ...
-  * Extensions for rclcpp: [system_modes](https://github.com/micro-ROS/system_modes/), [TF improvements](https://github.com/micro-ROS/geometry2), ...
+  * Extensions to rcl, i.e. the ROS 2 Client Library C API: [rcl_executor](https://github.com/micro-ROS/rcl_executor), [rclc](https://github.com/micro-ROS/rclc)...
```
Comment (ralph-lange) [inline _docs/overview/docker_ci_status/index.md:40]:
```suggestion
  * Extensions to rcl, i.e. the ROS 2 Client Library C API: [rclc](https://github.com/micro-ROS/rclc)
```

**Review by ralph-lange — CHANGES_REQUESTED** [review]

**Comment by pablogs9** [comment]
Reorder proposal:

1. CI Status for Tooling and Website
  Add here issues and PR counters, maybe the webpage repo should be removed from here.
2. CI Status for Source Code Repositories
3. CI Status for Example Application Repositories
4. Docker Images Status


Aditionally I will remove the "CI Status" since we are also showing issues and PR. Just "Status"?

**Comment by jamoralp** [comment]
> Reorder proposal:
> 
> 1. CI Status for Tooling and Website
>    Add here issues and PR counters, maybe the webpage repo should be removed from here.
> 2. CI Status for Source Code Repositories
> 3. CI Status for Example Application Repositories
> 4. Docker Images Status
> 
> Additionally I will remove the "CI Status" since we are also showing issues and PR. Just "Status"?

Ok, the order sounds good for me. I also believe the Docker section is not so-to-say "relevant" and can be moved to a less important spot in the resulting webpage.

**Comment by jamoralp** [comment]
@pablogs9 @FranFin @ralph-lange I reworked the section completely, and unified the tables' contents. Please re-review.

**Review by pablogs9 — DISMISSED** [review]

**Review by pablogs9 — APPROVED** [review]

**Review by ralph-lange — APPROVED** [review]

**Pull request merged**